microsoft/PowerToys
# [PowerRename] Sort files

## Summary of the Pull Request

**What is this about:**
Order of files shown in PowerRename was dependent on which file received right-click (explained here https://github.com/microsoft/PowerToys/issues/11640#issuecomment-908997395)

With this change, files are always sorted by file name (following the same rules as File Explorer - directories on top, etc..)

**How does someone test / validate:**
1) Install PT v0.51.1
   - Select multiple files and open PowerRename
   - Observe that files start with file which received right click
2) Install PT using this branch
   - Repeat the steps and observe that files are sorted in list view regardless which file received right-click

## Quality Checklist

- [x] **Linked issue:** #14288

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
